packages/protocol/contracts-0.8/governance/Validators.sol

```diff
@@ -613,13 +644,26 @@ contract Validators is
     require(isValidatorGroup(account), "Not a validator group");
     ValidatorGroup storage group = groups[account];
     group.slashInfo.multiplier = FixidityLib.wrap(group.slashInfo.multiplier.unwrap().div(2));
-    group.slashInfo.lastSlashed = now;
+    group.slashInfo.lastSlashed = block.timestamp;
+  }
+
+  /**
+   * @notice Allows the EpochManager contract to mint stable token for itself.
+   * @param amount The amount to be minted.
+   */
+  function mintStableToEpochManager(
```

Comment:
as said earlier i think this is unnecessary delegation just have this logic on the epoch manager

Reply:
The original approach was to have the logic in `epochManager` , but for this to work, we would need mento to update their contract to allow `epochManager` to mint. Since they have a different deployment/development process, this would add more time to release.

Reply:
ok, then please add a comment here, explaining this, so we can change it in the future

Reply:
addressed by https://github.com/celo-org/celo-monorepo/pull/11223 commit 66c4a89542ac3b03a3dad2fadcac803f7bba3205